src/components/Icons/Information/InfoErrorCircle.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
```

Comment:
🔧  Já sabe né, hehehe
```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill, getSize, getViewBox } from '../utils';
import type { IconProps } from '../types';
```

Reply:
comment resolved in https://github.com/magnetis/astro-native/pull/565/commits/7430dbdeb25fa06af4dcf3f8522f4cd0caddf050